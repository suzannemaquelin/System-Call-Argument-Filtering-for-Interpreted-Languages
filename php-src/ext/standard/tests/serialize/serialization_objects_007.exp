in __autoload(FOO)
in check(FOO)
in __autoload(FOO)

Warning: unserialize(): Function check() hasn't defined the class it was called for in %s on line 23
object(__PHP_Incomplete_Class)#%d (1) {
  ["__PHP_Incomplete_Class_Name"]=>
  string(3) "FOO"
}
Done