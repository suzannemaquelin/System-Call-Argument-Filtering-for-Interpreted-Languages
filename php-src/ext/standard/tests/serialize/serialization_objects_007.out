Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
in __autoload(FOO)
in check(FOO)
in __autoload(FOO)

Warning: unserialize(): Function check() hasn't defined the class it was called for in /root/php-src/ext/standard/tests/serialize/serialization_objects_007.php on line 23
object(__PHP_Incomplete_Class)#1 (1) {
  ["__PHP_Incomplete_Class_Name"]=>
  string(3) "FOO"
}
Done